engines/e_loader_attic.c

```diff
@@ -907,7 +986,7 @@ static OSSL_STORE_LOADER_CTX *file_open_with_libctx
 #ifdef _WIN32
         /* Windows file: URIs with a drive letter start with a / */
         if (p[0] == '/' && p[2] == ':' && p[3] == '/') {
-            char c = ossl_tolower(p[1]);
+            char c = tolower(p[1]);
```

Comment:
Because `ossl_tolower` is strictly libcrypto internal, unexported, and should stay that way.  This is now an external engine...
`tolower()` is C89, so this is not likely to break anything.

Reply:
And windows machines are probably not going to be worrying about EBCDIC, either...